extensions/amp-base-carousel/1.0/base-carousel.type.js

```diff
@@ -73,6 +75,15 @@ BaseCarouselDef.ScrollerProps;
  */
 BaseCarouselDef.SlideProps;
 
+/**
+ * @typedef {{
+ *   start: number,
+ *   end: number,
+ *   length: number,
+ * }}
+ */
+BaseCarouselDef.DimensionDef;
```

Comment:
Does this need to be a public type? I think it's only used internally in the implementation?

Reply:
It's only used internally, I'll defer to use the same `DimensionDef` defined in `0.1/dimensions`